samples/asgardeo-express-app/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "asgardeo-express-sample",
```

Comment:
```suggestion
  "name": "asgardeo-express-app",
```
Check other sample apps to check if this is correct.

Reply:
Should be `@asgardeo/express-app`